rxbinding/src/androidTest/java/com/jakewharton/rxbinding/RxBindingTestRunner.java

```diff
@@ -0,0 +1,34 @@
+package com.jakewharton.rxbinding;
+
+import android.app.KeyguardManager;
+import android.content.Context;
+import android.os.PowerManager;
+import android.support.test.runner.AndroidJUnitRunner;
+
+import static android.content.Context.KEYGUARD_SERVICE;
+import static android.content.Context.POWER_SERVICE;
+import static android.os.PowerManager.ACQUIRE_CAUSES_WAKEUP;
+import static android.os.PowerManager.FULL_WAKE_LOCK;
+import static android.os.PowerManager.ON_AFTER_RELEASE;
+
+public class RxBindingTestRunner extends AndroidJUnitRunner {
+  @Override public void onStart() {
+    runOnMainSync(new Runnable() {
```

Comment:
Does it work without this? I'm not sure why we originally added it, but tests won't run until `super.onStart()` is called so there's no need to post to the main thread synchronously.